audioclips/src/main/java/org/odk/collect/audioclips/AudioClipViewModel.kt

```diff
@@ -0,0 +1,203 @@
+package org.odk.collect.audioclips
+
+import android.media.MediaPlayer
+import androidx.lifecycle.*
```

Comment:
Right we're not getting style checks here but I'll rebase on top of `master` which will give us that setup.

Reply:
Nope you're right it also won't work because Kotlin. It looks like PMD support is coming but not quire there yet. I'll have a look at Kotlin specific projects for the moment to see if we can get a few of our rules applied to Kotlin so we're not too far behind when PMD supports both.